SETUP.md

```diff
@@ -6,31 +6,50 @@ This assumes you are using VSCode as your IDE. If you have errors or issues, you
 
 ### Discord Bot Account
 
-1. Create a discord bot account, and have the application ID and bot token saved.
+1. Create a discord bot account in the [Discord Developer Portal](https://discord.com/developers/applications).
+   - Remember the **Application ID** & **Token**. Both are used later in **Configuration**.
+   - If you don't save your Token you will have to reset it for a new one.
 2. Ensure your bot has `Privileged Gateway Intents > Server Members Intent` enabled.
 
 ### Environment
 
 1. Install [NvM](https://github.com/coreybutler/nvm-windows/), then use it to install NodeJS v20.15.0 OR install the nodejs version directly.
-2. Install [Postgres 16](https://www.postgresql.org/download/) and PGAdmin4 for interacting with postgres (optional, but helpful)
-3. Install Yarn using: `npm i -g yarn`
-4. Clone the repo: `git clone https://github.com/oldschoolgg/oldschoolbot.git`
-5. Run the following commands in the root of the repo: `corepack enable`, `yarn`, `npx prisma db push` and `npx prisma db push --schema ./prisma/robochimp.prisma`
+2. Install [Python](https://www.python.org/downloads/) used for multiple dependencies.
+3. Install [Visual Studio Installer](https://visualstudio.microsoft.com/downloads/).
+   - Install the `Community` version.
+   - Once downloaded select and install `Workloads > Desktop development with C++`.
```

Comment:
Are these really needed? I believe the nodejs installer installs the necessary build tools.

Reply:
I can't say with 100% certainty but in two different instances helping others they showed me this:
![Discord_iVuJLG7Wp2](https://github.com/user-attachments/assets/9e915980-291e-4d21-a259-593460b19b79)

Both instances they couldn't get this to clear without windows build tools and python.